Integrations/src/test/java/io/deephaven/integrations/learn/ComputerTest.java

```diff
@@ -23,93 +22,99 @@ public static void createTable() {
+        Function<Object[], Object> modelFunc = (params) ->
+        {
+            return 1;
+        };
+
+        Computer computer = new Computer(null, modelFunc, inputs, batchSize);
     }
 
     @Test(expected = io.deephaven.base.verify.RequirementFailure.class)
     public void nullFunctionTest() {
 
-        final Function<Object[], Object> func = null;
+        final Input[] inputs = createInputs(args -> args);
         final int batchSize = 7;
 
-        Computer computer = new Computer(table, func, inputs, batchSize);
+        Function<Object[], Object> modelFunc = null;
+
+        Computer computer = new Computer(table, modelFunc, inputs, batchSize);
     }
 
     @Test(expected = io.deephaven.base.verify.RequirementFailure.class)
     public void invalidBatchSizeTest() {
 
-        final Function<Object[], Object> func = args -> args;
+        final Input[] inputs = createInputs(args -> args);
         final int batchSize = 0;
 
-        Computer computer = new Computer(table, func, inputs, batchSize);
+        Function<Object[], Object> modelFunc = (params) ->
+        {
+            return 1;
+        };
+
+        Computer computer = new Computer(table, modelFunc, inputs, batchSize);
     }
 
     @Test
     public void computeGatherGetTest() {
 
-        final int batchSize = 7;
 
+        final Input[] inputs = createInputs(args -> args);
+
+        // initialize necessary reference objects and batchSize parameter
+        final int batchSize = 7;
         final IndexSet[] indexSetTarget = new IndexSet[1];
-        ColumnSource<?>[][] colSourceTarget = new ColumnSource[inputs.length][];
+        final ColumnSource<?>[][] colSourceTarget = new ColumnSource[inputs.length][];
+
+        // create columnSources based off of given Input and tables
         for (int i = 0 ; i < inputs.length ; i++) {
             colSourceTarget[i] = inputs[i].createColumnSource(table);
         }
 
-        final int[] counter = {0};
-
-        Function<Object[], Object> myGather = (params) ->
+        // define gather functions
+        Function<Object[], Object> myGather1 = (params) ->
         {
-            // first we test equality on IndexSet
+            Assert.assertEquals(2, params.length);
             Assert.assertEquals(indexSetTarget[0], params[0]);
-
-            // now we test equality on ColumnSources, which depends on which input object we are comparing
-            if (counter[0] == 0) {
-
-                Assert.assertTrue(Objects.deepEquals(new Object[]{colSourceTarget[0]}[0], params[1]));
-                counter[0]++;
-
-            } else {
-                Assert.assertTrue(Objects.deepEquals(new Object[]{colSourceTarget[1]}[0], params[1]));
-            }
+            Assert.assertTrue(Objects.deepEquals(new Object[]{colSourceTarget[0]}[0], params[1]));
 
             return 4;
         };
 
-        Function<Object[], Object> myModel = (thisInput) ->
+        Function<Object[], Object> myGather2 = (params) ->
         {
```

Comment:
Move these squiggles for inline funcs to the line above.  Same comment everywhere.